test/Business_Logic/MobiusDAO/README.md

```diff
@@ -0,0 +1,99 @@
+# MobiusDAO
+
+- **Type:** Exploit
+- **Network:** BSC
+- **Total lost:** ~2.15MM
+- **Category:** Bad Arithmetic
+- **Vulnerable contracts:**
+- - [Exploited Contract](https://bscscan.com/address/0x95e92b09b89cf31fa9f1eca4109a85f88eb08531) (Unverified Implementation)
+- **Tokens Lost**
+- - ~9.7 Quadrillion MBU
+
+- **Attack transactions:**
+- - [Attack Tx](https://bscscan.com/tx/0x2a65254b41b42f39331a0bcc9f893518d6b106e80d9a476b8ca3816325f4a150)
+
+- - Deployer EOA: [0xB32A53Af96F7735D47F4b76C525BD5Eb02B42600](https://bscscan.com/address/0xB32A53Af96F7735D47F4b76C525BD5Eb02B42600)
+
+- **Attack Block:**: 49470430
+- **Date:** May 11, 2025
+- **Reproduce:** `forge test --match-contract Exploit_MobiusDAO -vvv`
+
+## Step-by-step Overview
+
+1. Set up a malicious contract
+2. Call MobiusDAO's `deposit` function through the exploit contract with 0.001 WBNB
+3. Receive over 9.7 quadrillion tokens
+4. Swap the tokens for USDT using PancakeSwapV2
+
+## Detailed Description
+
+Decimal precision is critical when performing arithmetic operations between tokens in smart contracts. In the case of MobiusDAO, a miscalculation involving decimals resulted in a severe vulnerability that ultimately led to a loss of approximately $2.15 million. The contract involved was not verified on-chain and, based on its behavior, was likely not audited either.
```

Comment:
Is decimal precision relevant here, or just literally the contract multiplying a price when it shouldn't?